docs/technical-overview/stack.md

```diff
@@ -17,8 +17,7 @@ For the Dev.to tech stack we use:
 - [_Sidekiq_](https://github.com/mperham/sidekiq) and
   [_Active Job_](https://guides.rubyonrails.org/active_job_basics.html) for
   background workers
-- [_Algolia_](https://www.algolia.com/) for search (Algolia will be phased out
-  in favor of [_Elasticsearch_](https://www.elastic.co/))
+- [_Elasticsearch_](https://www.elastic.co/) for in app searching
```

Comment:
maybe we can link https://www.elastic.co/elasticsearch/ instead of the company's homepage